Modules/System/Upgrade Tags/src/UpgradeTagImpl.Codeunit.al

```diff
@@ -29,6 +29,31 @@ codeunit 9996 "Upgrade Tag Impl."
         SetUpgradeTagForCompany(NewTag, CopyStr(CompanyName(), 1, MaxStrLen(ConstUpgradeTags.Company)));
     end;
 
+    procedure SetSkippedUpgrade(ExistingTag: Code[250]; SkipUpgrade: Boolean): Boolean
+    begin
+        exit(SetSkippedUpgrade(ExistingTag, CompanyName(), SkipUpgrade));
+    end;
+
+    procedure SetSkippedUpgrade(ExistingTag: Code[250]; TagCompanyName: Code[30]; SkipUpgrade: Boolean): Boolean
+    var
+        UpgradeTags: Record "Upgrade Tags";
+    begin
+        if UpgradeTags.Get(ExistingTag, TagCompanyName) then begin
+            UpgradeTags."Skipped Upgrade" := SkipUpgrade;
```

Comment:
It would be good to check if value is different so we don't fire blank modify statements

Reply:
Added the condition